meta-intel-edison-distro/conf/distro/poky-edison.conf

```diff
@@ -6,12 +6,12 @@ PREFERRED_VERSION_linux-yocto = "5.%"
 PREFERRED_PROVIDER_virtual/bootloader ?=  "u-boot"
 PREFERRED_VERSION_u-boot ?= "2019.04%"
 PREFERRED_VERSION_u-boot-fw-utils ?= "2019.04%"
-PREFERRED_VERSION_nodejs = "8.%"
-PREFERRED_VERSION_nodejs-native = "8.%"
+PREFERRED_VERSION_nodejs = "10.%"
+PREFERRED_VERSION_nodejs-native = "10.%"
 BINDINGS_pn-mraa="python nodejs"
 BINDINGS_pn-upm="python nodejs"
```

Comment:
But on warrior (nodejs 10) upm enabling bindings cause an error due to IdleNotification deprecated and I don't know how to fix. @alext-mkrs ?